docs/node-operators/migration-guides/upgrade-node.md

```diff
@@ -1,40 +1,195 @@
+
+### When should I upgrade config?
+
+You can update the config any time before the upgrade block happens.
+
+We recommend you do it in the following way:
+
+1. Make a copy of the original config.
+2. Update the config in the copied files.
+3. A few hours before the upgrade, replace the original config with updated one. 
+
+:::caution Back up your original config files
+Make sure you have a backup for the original config in case you need them in the future.
+:::
+
+We do not recommend you do it too early in case you have to restart your node with the current version for some reason, for example if your node fails or your server restarts, etc.
+
+### Config changes
+
+The default locations for configuration files:
+
+- data-node: `YOUR_VEGA_HOME/config/data-node/config.toml`
+- vega-core: `YOUR_VEGA_HOME/config/node/config.toml`
+
+Add the following new parameters to the configuration files.
+
+We recommend checking all of the changes on your own. Follow the below instructions to do it, and read through every description to understand the changes:
+
+1. Download vega `v0.73.0`
+2. Generate config files within the temp home directory
+  - vega config
+  -  tendermint config
+  - data-node config
+3. Compare the new generated file in the temp location and the old file to see the differences.
+
+#### `EvtForward.EthCall`
+
+- `config file`: vega-core
+- `description`: The parameter allows you to define basic configurations like the call period to ethereum node.
+- `kind`: new parameter
+
+```toml title="YOUR_VEGA_HOME/config/node/config.toml"
+[EvtForward]
+  [EvtForward.EthCall]
+    Level = "Info"
+    PollEvery = "20s"
+```
+
+#### `Vesting`
+
+- `config file`: vega-core
+- `description`: Manipulate the vesting engine configuration
+- `kind`: new parameter
+
+```toml title="YOUR_VEGA_HOME/config/node/config.toml"
+[Vesting]
+  Level = "Info"
+```
+
+#### `Broker.Socket.DialTimeout`
+
+- `config file`: vega-core
+- `description`: The below change(`Broker.Socket.DialTimeout`) is fundamental when you run the data node, but it also needs to be updated for a validator node. We recommend you update it to `96 hours` due to changes in the data node initialization mechanism. It now downloads and loads all the segments into the database by default. This process may take a day or more.
+- `kind`: parameter change
+
+```toml title="YOUR_VEGA_HOME/config/node/config.toml"
+[Broker]
+  [Broker.Socket]
+    DialTimeout = "96h0m0s"
+```
+
+#### `Snapshot.StartHeight`
+
+- `config file`: vega-core
+- `description`: There is a change in the snapshot mechanism. The default value for the `Snapshot.StartHeight` parameter changed from `-1` to `0`. However, we **DO NOT** recommend changing this parameter now for existing nodes! Leave the `-1`, because your node can be destroyed when you start your node with the wrong binary when you have `Snapshot.StartHeight = 0`. For the new logic to load a snapshot, see the below pseudocode block.
```

Comment:
I think we can provide a bit more explanation and be a little be less dramatic, because it sounds a bit like we did not think this through:

```suggestion
- `description`: The behavior of the parameter `Snapshot.StartHeight` changed, and its default value has been updated from `-1` to `0`. **However, we DO NOT recommend changing this parameter before successfully migrating to version 0.73.0!**. Prior 0.73, setting this parameter to `0` triggers the removal of the existing snapshots. If you start with the wrong binary or end up with a rollback, your node will use the old behavior and remove existing snapshots. For the new logic to load a snapshot, see the below pseudocode block.
```